artifacts.go

```diff
@@ -231,13 +227,13 @@ func readArtifact(c *cli.Context) error {
 func getKey(path string) ([]byte, error) {
 	f, err := os.Open(path)
 	if err != nil {
-		return nil, errors.New("Invialid key path.")
+		return nil, fmt.Errorf("Invalid key path: %s", path)
```

Comment:
Nice. This I did not know. Usually I've been using `github.com/pkg/errors` package if I needed errors formatting.